venues/auai.org/UAI/2018/webfield/bidWebfield_updated.js

```diff
@@ -391,7 +391,7 @@ function renderContent(validNotes, tagInvitations, metadataNotesMap) {
             }
             Webfield.ui.searchResults(sortedNotes, submissionListOptions);
           } else {
-            Webfield.ui.searchResults(notes, submissionListOptions);
+            Webfield.ui.searchResults(_.sortBy(notes, n => n.content.title.toLowerCase()), submissionListOptions);
```

Comment:
This will cause problems in for any users of IE. Arrow functions should only be used on the server side.